sympy/solvers/solveset.py

```diff
@@ -278,17 +280,19 @@ def _is_function_class_equation(func_class, f, symbol):
     ========
 
     >>> from sympy.solvers.solveset import _is_function_class_equation
-    >>> from sympy import C, tan, sin, tanh, sinh, exp
```

Comment:
Good catch. C should never be necessary in a docstring.
Did you run the doctests? (Don't worry if not - Travis will do doctesting anyway.)